examples/features/deadline/server/main.go

```diff
@@ -16,7 +16,8 @@
  *
  */
 
-// Binary server is an example server.
+// Binary server demonstrates how to handle RPCs with deadlines and propagate
```

Comment:
Same here.

// Binary server demonstrates how to handle RPCs with deadlines and how to propagate
// deadlines in requests.

Reply:
Done